src/libs/ReportUtils.js

```diff
@@ -1656,7 +1679,8 @@ function getReportPreviewMessage(report, reportAction = {}, shouldConsiderReceip
         }
     }
 
-    if (isSettled(report.reportID)) {
+    // Show Paid preview message if it's settled or if the amount is paid & stuck at receivers end for only chat reports.
+    if (isSettled(report.reportID) || (report.isWaitingOnBankAccount && !isPreviewMessageForParentChatReport)) {
```

Comment:
@situchan @techievivek review this please. Updated to resolve issue with `isSettled` function update.

Reply:
Would it not show as settled for the iou report now?

Reply:
That's a or condition so it would show paid for iou as well.

Reply:
That's what I am saying then how would it show system message in case of iou report.